### PR TITLE
Adding neurodamus-mousify package

### PR DESCRIPTION
**Pull request by ferdonline**

### CODE REVIEW

File: var/spack/repos/builtin/packages/neurodamus-core/package.py [hunk 1]
```diff
@@ -12,7 +12,8 @@ class NeurodamusCore(Package):
     git      = "ssh://bbpcode.epfl.ch/sim/neurodamus-core"
 
     version('develop', git=git, branch='master')
-    version('2.2.1', git=git, tag='2.2.1', preferred=True)
+    version('2.3.0', git=git, tag='2.3.0', preferred=True)
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/neurodamus-core/package.py:15]: what is changed in the code? just curious..

**Review by pramodk — APPROVED** [review]
LGTM

**Comment by ferdonline** [comment]
please retest

File: var/spack/repos/builtin/packages/neurodamus-core/package.py (hunk 1 above)
Comment (ferdonline) [reply var/spack/repos/builtin/packages/neurodamus-core/package.py:15]: It's the version with the different handling of ModOverride

**Comment by ferdonline** [comment]
please retest

**Pull request merged**